resources/data/Randomizer/doors.json

```
@@ -3523,13 +3523,14 @@
 		"Id": "D06Z01S08[E]",
 		"Direction": 2,
 		"OriginalDoor": "D06Z01S14[W]",
-		"Logic": "D06Z01S08[N] || D06Z01S08[E] || wallClimb"
+		"Logic": "D06Z01S08[N] || D06Z01S08[E] || wallClimb || doubleJump && canEnemyBounce"
 	},
 	{
 		"Id": "D06Z01S08[N]",
 		"Direction": 0,
 		"OriginalDoor": "D06Z01S13[S]",
-		"VisibilityFlags": "ThisDoor"
+		"VisibilityFlags": "ThisDoor",
```

```suggestion
		"VisibilityFlags": "ThisDoor, EnemySkipsAndDoubleJump",
```
Since there is now a way that this door can be reached other than by itself, we need to add a new visibility flag since it can now also be reached if enemy skips are allowed and double jump is shuffled.  It still can't normally be reached in vanilla though.